### ISSUE TITLE
For Cycle 4

### ISSUE THREAD

**Issue by juantellez** [bisq-network/compensation#318]
## Summary
 - BSQ requested: 553 BSQ

### Translations and reviews for Bisq-Desktop
Reviews: 11,976 words x 0.035 BSQ = 419.16 BSQ

### Translations and reviews for Bisq-Website
Reviews for dao_content.yml: 405 words x 0.035 BSQ = 14.17 BSQ
Reviews for images.yml: 31 words x 0.035 BSQ = 1.08 BSQ
Reviews for main_nav.yml: 18 words x 0.035 BSQ = 0.63 BSQ
Reviews for strings.yml: 3,374 words * 0.035 BSQ = 118.09 BSQ

## Contributions in progress

Awaiting for Bisq-Website translation be completed to continiue the review

## Roles performed
NONE

## Screenshots from transifex
![dao_content_yml](https://user-images.githubusercontent.com/6751110/61751574-6b512a80-ad6e-11e9-97c6-7ddd3cfcf631.png)
![images_yml](https://user-images.githubusercontent.com/6751110/61751575-6be9c100-ad6e-11e9-9f91-d140bab0421e.png)
![main_nav_yml](https://user-images.githubusercontent.com/6751110/61751576-6b512a80-ad6e-11e9-832a-248ce917e056.png)
![revision-aplicacion](https://user-images.githubusercontent.com/6751110/61751577-6be9c100-ad6e-11e9-9f55-2947dd049955.png)
![strings_yml](https://user-images.githubusercontent.com/6751110/61751578-6be9c100-ad6e-11e9-908c-32431f0e7a9e.png)
![traduccion-aplicacion](https://user-images.githubusercontent.com/6751110/61751580-6be9c100-ad6e-11e9-8be5-f88457dd8463.png)

**Comment by juantellez** [comment]
Hi guys, this my first compensation request, I hope it is prepared correctly, otherwise please let me know if something needs to be specified or justified.
Thanks in advance

**Comment by y3v63n** [comment]
Hi @juantellez! Great job with reviews and translations! All looks good, but you no longer have to indicate your BSQ address here. Also, a quick question. Are you sure you translated over 6k words in the Bisq Desktop project? I was under impression that it was almost completely done by another translator. Could it be that you edited previously reviewed translations? In that case, Transifex filters count edits as new translations and it is better to go the Bisq Desctop dashboard in Transifex and check your localization activity there.

**Comment by juantellez** [comment]
> Hi @juantellez! Great job with reviews and translations! All looks good, but you no longer have to indicate your BSQ address here. Also, a quick question. Are you sure you translated over 6k words in the Bisq Desktop project? I was under impression that it was almost completely done by another translator. Could it be that you edited previously reviewed translations? In that case, Transifex filters count edits as new translations and it is better to go the Bisq Desktop dashboard in Transifex and check your localization activity there.

Hi y3v63n, i have checked the Dashboard as you sugested and get surprised to see that i has made 0 translations, i maeby crazy but i remember make a lot of translations the first time i get into transifex, then when i was set as a reviewer i made only reviews. So i have made a count of all translations made by each translator using the filters and that make sense:

The total words for the spanish translation of the Bisq-Desktop is 16,564 words. Below is the amount of word translated by each translator:

tasmanoide: 8687
juantellez: 6320
frustramatic: 118
r4wl: 825
cfair: 51
ripcurlx: 81
rudygodoy: 451
yield: 31
**Total: 16.564**

I don't know why the dashboard shows i have made no translations. I even made a couple of translations in the Bisq-Website right now and also are not counted at the bisq-website dashboard.

Hope this can be clarified and would be great to know why the dashboard don't show any translation made from me.

**Comment by juantellez** [comment]
By the way, i have notices that my first count of words for bisq-desktop was 6,346 when actually where 6320. I have already made the correction.

**Comment by MwithM** [comment]
The dashboard displays translations made in that signaled month. In this image, July. 

![imagen](https://user-images.githubusercontent.com/50149324/61865492-986b1d80-aed3-11e9-9d4c-31aaaa5b3d43.png)

When you edit an already translated string it counts as the whole translation has been made by the user who edited the string, even if it's only minor changes. 6320 words translated (more than a third of the desktop project) makes sense only after all your reviews, not before, because you had to edit a lot of strings while reviewing. This is an example of a string that was already translated and, when edited for review, it counts as translated by you.

![imagen](https://user-images.githubusercontent.com/50149324/61866044-ba18d480-aed4-11e9-91e4-d6a347ee9a4c.png)

The reason why now the word count for reviewed strings on Desktop is 6320 is because I overreviewed the string #418.

![imagen](https://user-images.githubusercontent.com/50149324/61866632-cea99c80-aed5-11e9-9116-442cc33cecd4.png)

**Comment by y3v63n** [comment]
@juantellez I tend to agree with @MwithM. 

You should try and check dashboard stats for each month separately. 

You joined the team in late April, and if you check our [monthly reports](https://github.com/bisq-network/roles/issues/20), by the end of April Spanish had just [2 untranslated](https://github.com/bisq-network/roles/issues/20#issuecomment-487948127) strings out of the total 16,200 words, and in May another ~300 words were translated. Before that I can see that in November 2018, there had already been over [13k words](https://github.com/bisq-network/roles/issues/20#issuecomment-443166842) translated, meaning that there is no chance you could translate those 6k words.

**Comment by goo-technology** [comment]
Ok, i see. Then i have no translations, just reviews. 
I tried to edit the issue again to remove the amount corresponding to translations but I no longer see the option to edit. An admin have to remove it or close so i can pos a new issue with the corrections?

**Comment by MwithM** [comment]
> 
> 
> Ok, i see. Then i have no translations, just reviews.
> I tried to edit the issue again to remove the amount corresponding to translations but I no longer see the option to edit. An admin have to remove it or close so i can pos a new issue with the corrections?

I assume it's because you are using different user accounts.

**Comment by juantellez** [comment]
> 
> 
> > Ok, i see. Then i have no translations, just reviews.
> > I tried to edit the issue again to remove the amount corresponding to translations but I no longer see the option to edit. An admin have to remove it or close so i can pos a new issue with the corrections?
> 
> I assume it's because you are using different user accounts.

You're right.

Issue is updated.

**Comment by MwithM** [comment]
Closed as accepted
